man/systemd-run.xml

```diff
@@ -274,6 +274,15 @@
         "hello" >&amp;2</command> instead, which is mostly equivalent and avoids this pitfall.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--shell</option></term>
+        <term><option>-S</option></term>
+
+        <listitem><para>A shortcut for <literal>--pty --same-dir --wait --collect --service-type=exec $SHELL</literal>,
+        i.e. requests an interactive shell in the current working directory, running in service context, accessible
+        with a single switch.</para></listitem>
```

Comment:
Many people think `$SHELL` is the current shell. So just to avoid confusion, maybe add a note here:
`"Note that $SHELL refers to the user's login shell, and not necessarily the one that is currently used."`

Reply:
actually it *is* `$SHELL` that is used here. My thinking was that if you are a zsh user then `systemd-run -S` should get you a zsh, even if if the destination user doesn't have zsh set. i.e. the switch shall inherit pty, cwd and shell from the caller...

Reply:
I didn't mean the destination user, I meant the user starting running the `systemd-run` command.
```console
$ echo $SHELL
/bin/bash
$ zsh
% echo $SHELL
/bin/bash
```
There is some blurriness, because I see that `bash` actually sets `$SHELL` itself now, but in general, `$SHELL` is the original login shell.